addon/objects/uml-primitives/fd-uml-transition.js

```
@@ -61,6 +61,70 @@ export let Transition = Connection.define('flexberry.uml.Transition', {
+      .join('\n');
+
+    $input.val(updatedDescription);
+    this.updateInputWidth('.description-input');
+  },
+
+  updateInputPosition(index, selector) {
+    let position = this.getLabelCoordinates(this.model.label(index).position);
+    let textarea = this.$box.find(selector)[0];
+    let textWidth = textarea.scrollWidth;
+
+    $(this.$box.find(selector)).css({
```

ESLint

[error] Do not use global `$` or `jQuery`

(ember/no-global-jquery)